dev/archery/archery/docker/core.py

```diff
@@ -43,6 +44,10 @@ def flatten(node, parents=None):
         raise TypeError(node)
 
 
+def verbosity_args():
+    return ['--quiet'] if log_ctx.quiet else []
```

Comment:
Hmm. I don't want to use `archer --quiet ...` in many places. We will miss `--quiet` easily in a new CI configuration.

How about checking `GITHUB_ACTIONS=true` environment variable?
See also: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables

```suggestion
    return ['--quiet'] if os.environ.get('GITHUB_ACTIONS') == 'true' else []
```

Reply:
Perhaps a general `running_on_ci()` function?

Reply:
It's a good idea.